charts/posthog/ALL_VALUES.md

```diff
@@ -26,6 +26,7 @@ The following table lists the configurable parameters of the PostHog chart and t
 | events.hpa.cputhreshold | int | `60` | CPU threshold percent for the events stack HorizontalPodAutoscaler. |
 | events.hpa.minpods | int | `1` | Min pods for the events stack HorizontalPodAutoscaler. |
 | events.hpa.maxpods | int | `10` | Max pods for the events stack HorizontalPodAutoscaler. |
+| events.securityContext.enabled | bool | `false` | SecurityContext for events. |
```

Comment:
This file is autogenerated using the comments in the `values.yaml` file. Please don't manually edit it

Reply:
Done